不要タスク/リンクを削除する

# 目的 : Must
## 背景
#929 をマージしたら、imgビルドに失敗した

``` shell
/usr/bin/../lib/gcc/arm-none-eabi/4.9.3/../../../../arm-none-eabi/bin/ld: cfg2_out section `.data_shared' will not fit in region `ROM'
/usr/bin/../lib/gcc/arm-none-eabi/4.9.3/../../../../arm-none-eabi/bin/ld: region `ROM' overflowed by -5504 bytes
collect2: error: ld returned 1 exit status
```
# やる事 : Must

不要タスクとリンクを削除する
# 考慮した事
## [リスク](https://kotobank.jp/word/%E3%83%AA%E3%82%B9%E3%82%AF-183705)

なし
## [懸念事項](https://kotobank.jp/word/%E6%87%B8%E5%BF%B5-490895)

なし

--- a/hrp2/sdk/workspace/hackEV/Makefile.inc
+++ b/hrp2/sdk/workspace/hackEV/Makefile.inc
@@ -20,7 +20,6 @@ APPL_CXXOBJS += \
     utilities.o \
     Logger.o \
     SonarSensorController.o \
-    gyroboy.o \
     ColorSensorController.o \
     GyroSensorController.o \
     MotorWheel.o \

--- a/hrp2/sdk/workspace/hackEV/app.cfg
+++ b/hrp2/sdk/workspace/hackEV/app.cfg
@@ -8,19 +8,12 @@
 INCLUDE("app_common.cfg");
 
 #include "app.h"
-#include "pid_controller.h"
-#include "gyroboy.h"
 
 DOMAIN(TDOM_APP) {
-CRE_TSK(BALANCE_TASK, { TA_NULL, 0, balance_task, TMIN_APP_TPRI, STACK_SIZE, NULL });
 CRE_TSK(MAIN_TASK, { TA_ACT, 0, main_task, TMIN_APP_TPRI + 1, STACK_SIZE, NULL });
-CRE_TSK(PID_CONTROLLER_TASK, { TA_NULL, 0, pid_controller_task, TMIN_APP_TPRI + 2, STACK_SIZE, NULL });
-CRE_TSK(DRIVE_CONTROLLER_TASK, { TA_NULL, 0, drive_controller_task, TMIN_APP_TPRI + 3, STACK_SIZE, NULL });
 EV3_CRE_CYC(GYRO_UPDATE_TASK, { TA_STA, 0, gyro_update_task, 10, 0 });
 EV3_CRE_CYC(LOG_MONITORING_TASK, { TA_STA, 0, log_monitoring_task, 500, 0 });
 EV3_CRE_CYC(POSITION_UPDATE_TASK, { TA_STA, 0, position_update_task, 2, 0 });
-CRE_TSK(IDLE_TASK, { TA_NULL, 0, idle_task, TMIN_APP_TPRI + 4, STACK_SIZE, NULL });
 }
 
 ATT_MOD("app.o");
-ATT_MOD("utilities.o");

